opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -154,6 +190,14 @@ def _import_exporters(
         else:
             raise RuntimeError(f"{exporter_name} is not a trace exporter")
 
+    for (exporter_name, exporter_impl,) in _import_config_components(
+        metric_exporter_names, "opentelemetry_metrics_exporter"
```

Comment:
`opentelemetry_metrics_exporter` is only present for `ConsoleMetricExporter`. We should add it for the OTLP and OTLP/GRPC exporters too, since they support metrics export. (Although given code above that replaces `otlp` with `otlp_proto_grpc`, perhaps the entrypoints on the OTLP exporter are legacy?)

Reply:
This will be addressed in another PR.